templates/skeleton/app/root.tsx

```diff
@@ -103,10 +97,14 @@ export async function loader({context}: LoaderFunctionArgs) {
       cart: cartPromise,
       footer: footerPromise,
       header: await headerPromise,
-      isLoggedIn,
+      isLoggedInPromise,
```

Comment:
nit picky, but we don't name the other serialized variables with "Promise" in here.

Reply:
I added the post fix Promise to all the things that still need resolving